lib/pbench/server/api/resources/users_api.py

```diff
@@ -269,14 +268,22 @@ def post(self):
                         "message": "failure message"
                     }
         """
-        auth_token = self.auth.token_auth.get_auth()["token"]
-        user = self.auth.token_auth.current_user()
+        auth_token = self.auth.get_auth_token(self.logger)
+        user = Auth.verify_auth(auth_token=auth_token)
+
+        # "None" user represents that either the token is not present in our database or it is an expired token.
+        # Expired token is already deleted by now if we reach here.
+        # In either case we will return suceess to a client, since the same token can not be used again.
 
         try:
             ActiveTokens.delete(auth_token)
-            self.logger.info(
-                "Auth token entry deleted for user with username {}", user.email
-            )
+            # Do we need to log every time someone logs out
+            if user:
+                self.logger.debug("User {} logged out", user.username)
+            else:
+                self.logger.debug(
+                    "Logout: Token is not present in the db or user tried an expired token"
+                )
         except Exception:
             self.logger.exception("Exception occurred during deleting the user entry")
```

Comment:
So perhaps we should word this something like, `Exception occurred deleting an auth token`?